src/fragments/start/getting-started/reactnative/auth.mdx

```diff
@@ -0,0 +1,263 @@
+The next feature you will be adding is authentication.
+
+## Authentication with Amplify
+
+Amplify uses [Amazon Cognito](https://aws.amazon.com/cognito/) as the main authentication provider. Amazon Cognito is a robust user directory service that handles user registration, authentication, account recovery & other operations. In this tutorial, you'll learn how to add authentication to your application using Amazon Cognito and username/password login.
+
+## Create authentication service
+
+To add authentication to your app, run this command:
+
+```bash
+amplify add auth
+```
+
+Select the defaults for the following prompts:
+
+```console
+? Do you want to use the default authentication and security configuration? Default configuration
+Warning: you will not be able to edit these selections. 
+? How do you want users to be able to sign in? Username
+? Do you want to configure advanced settings? No, I am done.
+```
+
+To deploy the service, run the `push` command:
+
+```bash
+amplify push
+```
+
+Now, the authentication service has been deployed and you can start using it. To view the deployed services in your project at any time, go to Amplify Console by running the following command:
+
+```bash
+amplify console
+```
+
+## Create login UI
+
+Creating a login flow can be quite difficult and time consuming to get right. Luckily, Amplify UI has an [authentication](https://ui.docs.amplify.aws/react-native/connected-components/authenticator) component that provides an entire authentication flow for you, using the configuration you specified in **aws-exports.js.**
+
+### Install Amplify UI
+
+The `@aws-amplify/ui-react-native` package includes React Native specific UI components you'll use to build your app. Install it with the following command:
+
+<BlockSwitcher>
+<Block name="Expo CLI">
+
+```bash
+npm install @aws-amplify/ui-react-native react-native-safe-area-context react-native-get-random-values react-native-url-polyfill
```

Comment:
_Believe_ that `react-native-safe-area-context` is already included in the Expo SDK

Reply:
This seems to be true, just FYI it also has this as a dependency to install on the [UI docs for Expo ](https://ui.docs.amplify.aws/react-native/getting-started/installation)